### DISCUSSION TITLE
请问服务端是否有计划增加 socks 出站功能？

### DISCUSSION THREAD

**Discussion by chika0801** [juicity/juicity#27] [Q&A] [answered]
主要是方便 socks 出站到本机端口，再对接进入如 xray sing-box，按需进行一些分流。此功能 TUIC 作者是明确写在readme中不会增加。TUIC 那边问此功能的 ISS 还挺多的。

请问 juicity 是否有计划增加这项功能？

**Comment by chise0713** [comment, marked as answer]
把它当隧道用就不需要socks出站了吧
https://github.com/XTLS/Xray-core/issues/1844#issuecomment-1642049581

**Reply by chika0801** [reply]
你可以说，我正在用 Juicity 隧道，多高大上。23333。
